book/src/bridge-module/opaque-types/README.md

```diff
@@ -240,4 +240,45 @@ if val1 == val2 {
 } else {
     print("Not equal")
 }
+```
+
+#### #[swift_bridge(Hashable)]
+
+The `Hashable` attribute allows you to expose a Rust `Hash` trait via Swift's
```

Comment:
_very minor feedback_ -> `Rust's`

Reply:
Sorry.. actually can you make it `expose a Rust `Hash` trait implementation via ...`

Sorry for the hassle just trying to keep things somewhat consistent.

**You can ignore this feedback if you've already went and changed it to the first comment... Not a big deal for now...**

Reply:
>Sorry.. actually can you make it "... expose a Rust `Hash` trait implementation via ..."

Yes!

Reply:
I felt bad for nitpicking, glad you don't care.
Sweet, thanks!